code/modules/mob/living/simple_animal/guardian/types/standard.dm

```diff
@@ -15,9 +15,13 @@
 	set name = "Set Battlecry"
 	set category = "Guardian"
 	set desc = "Choose what you shout as you punch people."
-	var/input = stripped_input(src,"What do you want your battlecry to be? Max length of 6 characters.", ,"", 7)
```

Comment:
Why are you deleting this line you fool?

Also github doesn't support code suggestions on deleted lines so I can't fix this for you.